lib/generate.js

```diff
@@ -2,11 +2,20 @@ const fs = require('fs')
 const Mustache = require('mustache')
 const ncp = require('ncp').ncp
 const orgs = require('../out/data.json')
+const dates = require('../out/dates.json')
 
 const time = fs.statSync(`${__dirname}/../out/data.json`).mtime
+const today = new Date()
 const datetime = new Date(time).toUTCString()
 const rootURL = process.env.URL
 
+const competitionOpen = new Date(dates.competition_open_starts)
+const noClaims = new Date(dates.competition_open_ends)
+const totalTime = (noClaims - competitionOpen)/(24*3600*1000)
+const timeLeft = (noClaims-today)/(24*3600*1000)
+const daysLeft = parseInt(timeLeft)
+const percentagePassed = parseInt(100-(timeLeft/totalTime)*100)
```

Comment:
is `parseInt` even needed?

Reply:
Otherwise it would have so many digits, so maybe not readable?